Stack/Opc.Ua.Core/Stack/Tcp/TcpMessageType.cs

```diff
@@ -301,5 +301,14 @@ public static class TcpMessageLimits
         /// The certificates that have the key size larger than KeySizeExtraPadding need an extra padding byte in the transport message
         /// </summary>
         public const int KeySizeExtraPadding = 2048;
+
+        /// <summary>
+        /// Aligns the max message size to the nearest min buffer size.
+        /// </summary>
+        public static int AlignRoundMaxMessageSize(int value)
+        {
+            int alignmentMask = MinBufferSize - 1;
+            return (value + alignmentMask) & ~alignmentMask;
```

Comment:
would value + MinBufferSize - (value % MinBufferSize) be the same result or am I missing something here?

Reply:
I think not, since if MaxMessageSize % MinBufferSize == 0 than MaxMessageSize is allready alligned to MinBufferSize and your code will additionally add MinBufferSize.